language/move-prover/src/spec_translator.rs

```diff
@@ -1109,7 +1109,7 @@ impl<'env> SpecTranslator<'env> {
         match exp {
             Exp::Value(node_id, val) => {
                 self.set_writer_location(*node_id);
-                self.translate_value(*node_id, val);
+                self.translate_value(val);
```

Comment:
... I also pass information to functions even if its not needed right now. Say translate_value would need to report an error, then it needs node_id to get the location where to report the error...